packages/KSelect/KSelect.vue

```diff
@@ -435,7 +435,10 @@ export default {
             this.selectItems[i].selected = false
           }
 
-          this.selectItems[i].key = `${this.selectItems[i].label.replace(' ', '-').replace(/[^a-z0-9-_]/gi, '')}-${i}`
+          if (this.selectItems[i].label) {
+            this.selectItems[i].key = `${this.selectItems[i].label.replace(' ', '-')}-${i}`
+          }
```

Comment:
Each select item will need a unique key, so you'll need to add an `else` block or something here so that `key` is always defined. 

We do not want to utilize UUID or something similar as a fallback as this could cause issues with snapshot testing.

Reply:
Thanks for your comments! I have added generating a unique `key` in the easiest way 👍